doc/classes/ColorRect.xml

```diff
@@ -38,6 +38,11 @@
 	</methods>
 	<members>
 		<member name="color" type="Color" setter="set_frame_color" getter="get_frame_color">
+			The color in RGBA format.
+			[codeblock]
+			var col_rect = $ColorRect
+			col_rect.color = Color(1, 0, 0, 1) # Set color rect node to red
+			[/codeblock]
```

Comment:
I would simplify this example. The variable is unnecessary, and the comment is not accurate:

    $ColorRect.color = Color(1, 0, 0, 1) # Set ColorRect node's color to red